pages/view/contact/_id.vue

```diff
@@ -70,11 +70,9 @@
 
       <div class="flex justify-start mb-4">
         <div class="w-3/12 margins">
-          <nuxt-link :to="localePath('/search/contact')">
-            <AppButton custom_style="btn-cancel" data_cypress="formButton">
-              {{ $t('contact.back') }}
-            </AppButton>
-          </nuxt-link>
+          <AppButton @click="goBack" custom_style="btn-cancel" data_cypress="formButton">
```

Comment:
my bad I used the same data_cypress name for both buttons. each should have a different name

Reply:
Kk will fix now